### DISCUSSION TITLE
Is it possible to use only two version numbers, e.g. `1.23`

### DISCUSSION THREAD

**Discussion by lovetheguitar** [commitizen-tools/commitizen#867] [Q&A]
For some project I need to work with those version numbers. So I'd for example just let all non-breaking changes be a "feature" bump.

Thanks in advance! :)

**Comment by woile** [comment]
You can try to write you own version scheme: https://github.com/commitizen-tools/commitizen/blob/master/commitizen/version_schemes.py#L40
https://github.com/commitizen-tools/commitizen/blob/master/pyproject.toml#L94-L96

You need to create a package, follow the protocol, and then register it like in the second link

I think that could work
